tests/integration/tests.py

```diff
@@ -398,6 +398,25 @@ def test_protocol_v6(self):
 
         assert instance.message == 'hello'
 
+    def test_basic_quotas(self):
+        results = []
+        for n in range(10):
+            results.append(self._postWithHeader(
+                data={'message': 'hello'},
+                key=self.projectkey.public_key,
+                secret=self.projectkey.secret_key,
+                protocol='6',
+            ))
+
+        # 1 per hour
```

Comment:
Talked about this Friday, but this needs be the first thing to happen in this test